src/Pages/PopularRepositoriesTable/PopularRepositoriesTable.tsx

```diff
@@ -452,6 +468,25 @@ const PopularRepositoriesTable = () => {
               </FlexItem>
             </InputGroupItem>
           </InputGroup>
+          {(debouncedSearchValue !== '' || searchQuery !== '') && (
```

Comment:
Look into what debounced values are. 
Would it make sense to wait for the debouncedSearchValue to change this? 
Try typing something in the searchbox, and then hit clear filters. 
Does the chip immediately disappear?